### PR TITLE
Add travis check

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

Add a check to travis to find changed files (files apart of the PR) that have var/.* as an arguement.

> **This will fail your build if you change a file and do not remove all `var/` from proc arguements.**

## Why It's Good For The Game

Code standards.

### CODE REVIEW

**Comment by psykzz** [comment]
First to make sure it compiles when no changes detected.

**Comment by psykzz** [comment]
Now to add a failing test case.

**Comment by psykzz** [comment]
We actually need to fix our shit first before we test.

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1140,7 +1140,7 @@ will handle it, but:
 	if(final_x || final_y)
 		return locate(final_x, final_y, T.z)
 
-/proc/animate_speech_bubble(image/I, list/show_to, duration)
+/proc/animate_speech_bubble(image/I, list/show_to, var/duration)
```
Comment (Rohesie) [inline code/__HELPERS/unsorted.dm]:
```suggestion
/proc/animate_speech_bubble(image/I, list/show_to, duration)
```
Merge garbage.

Comment (psykzz) [reply code/__HELPERS/unsorted.dm]: (you didn't read did you)

Comment (Rohesie) [reply code/__HELPERS/unsorted.dm]: Ooops!

**Comment by spookydonut** [comment]
Should only apply to lines that are changed not the whole file

**Comment by spookydonut** [comment]
This matches `//      for(var/x in y)`

**Pull request merged**